.github/workflows/publish.yml

```diff
@@ -30,11 +29,36 @@ jobs:
         with:
           go-version-file: "go.mod"
           check-latest: true
-      - run: go version
+          cache: true
       - name: Run GoReleaser
         uses: goreleaser/goreleaser-action@v3
         with:
           version: latest
           args: release --rm-dist
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  releases-windows:
+    needs: tests
+    name: GoReleaser Build on Windows
+    if: startsWith(github.ref, 'refs/tags/')
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0 # See: https://goreleaser.com/ci/actions/
+      - uses: actions/setup-go@v3
+        with:
+          go-version-file: "go.mod"
+          check-latest: true
+          cache: true
```

Comment:
Out of curiosity, what is the use of this `cache` property?

Reply:
https://github.com/actions/setup-go#caching-dependency-files-and-build-outputs

> The action has a built-in functionality for caching and restoring go modules and build outputs.

It should speed up a bit the CI execution.